migrations/V246__Add_Key_Contact_Tables.sql

```diff
@@ -0,0 +1,68 @@
+ALTER TYPE TABLE_NAME ADD VALUE 'key_contact_category';
+
+-- Create table for key contact category
+CREATE TABLE IF NOT EXISTS  key_contact_category (
+    id UUID PRIMARY KEY NOT NULL,
+    category ZERO_STRING NOT NULL,
+
+    --META DATA
+    created_by UUID NOT NULL REFERENCES user_account(id),
+    created_dts TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    modified_by UUID REFERENCES user_account(id),
+    modified_dts TIMESTAMP WITH TIME ZONE
+);
+
+COMMENT ON TABLE key_contact_category IS
+'Table for storing key contact categories related to subject matter experts.';
+
+-- Ensure that each category has a unique name in key_contact_category
+ALTER TABLE key_contact_category
+ADD CONSTRAINT uniq_category UNIQUE(category);
+
+ALTER TYPE TABLE_NAME ADD VALUE 'key_contact';
+
+-- Create table for key contact
+CREATE TABLE IF NOT EXISTS  key_contact (
+    id UUID PRIMARY KEY NOT NULL,
+    user_id UUID REFERENCES user_account(id) ON DELETE CASCADE,
+
+    subject_area ZERO_STRING NOT NULL,
+    subject_category_id UUID NOT NULL REFERENCES key_contact_category(id) ON DELETE CASCADE,
+
+    mailbox_title ZERO_STRING NULL,
+    mailbox_address ZERO_STRING NULL,
```

Comment:
We don't do this elsewhere, but we might want to collate this column as well 
eg
```sql
COLLATE "case_insensitive", -- Set case-insensitive collation for the name column,  
```